### DISCUSSION TITLE
Projects (beta) - Table view columns are reset after exiting the project

### DISCUSSION THREAD

**Discussion by Artonus** [community/community#8837] [Projects and Issues]
I have added the Repository and Labels columns to the project in the Table View and changed their order . After this, I have exited the project and opened it again. The columns were in the state before the changes. Repository and Labels columns were hidden.

I very much like the idea of adding the idea for table view for the Project pages. Keep it up, I'm looking forward to it.
Regards
:)

**Comment by mvz** [comment]
I think you need to click 'Save Changes' in the upper right hand corner to keep adjustments to filters and columns.

**Comment by kemicky** [comment]
Contents of one view duplicate in the other views, very frustrating
